aws/components/network/setup.ftl

```diff
@@ -461,7 +511,7 @@
                         [#local forwardIpAddresses = getGroupCIDRs(ruleConfig.Source.IPAddressGroups, true, occurrence)]
                         [#local forwardPort = (ports[ruleConfig.Destination.Port])!{}]
                         [#local returnIpAddresses = [ "0.0.0.0/0" ]]
-                        [#local returnPort = (ports[ruleConfig.Destination.Port])!{}]
+                        [#local returnPort = (ports[ruleConfig.Source.Port])!{}]
```

Comment:
Was this a bug? Looks like a significant change not talked about in the description?

Reply:
- Fixes the port used for outbound rules in return traffic 

I don't think we've ever actually applied NACLs that this would have been noticed on.